apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.JobCrud.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright 2016 Google Inc. All Rights Reserved.
+// Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
I don't think so - basically the copyright year is typically the year the file was *first* generated.
(The changes in the first line in all these files is just about the BOM.)